html/admin/common/plugins/uppy.twig

```diff
@@ -85,11 +88,16 @@ x-amz-meta-size
                 { id: 'title', name: 'Title', placeholder: 'filetitle' },
                 { id: 'license', name: 'License', placeholder: 'specify license' },
                 { id: 'credit', name: 'Credit', placeholder: 'How should the image be credited? Don\'t include "Image:"' },
-                { id: 'ownwork', name: 'Photo from a member of the Nouse Team?', render: function({value, onChange}, h) {
+                { id: 'ownwork', name: 'Photo from a member of the TechSoc Team?', render: function({value, onChange}, h) {
                         return h('input', { type: 'checkbox', onChange: (ev) => onChange(ev.target.checked ? 'on' : 'off'), defaultChecked: value === 'off' })
                     } }
                 {% endif %}
             ]
+        }).use(UppyGoogleDrive, {
+            target: Uppy.Dashboard,
+            google_client_id: "{{ CONFIG["AUTH-PROVIDERS"]["GOOGLE"]["keys"]['id'] }}",
```

Comment:
@Jbithell Would you like to use the same google key as the login or a different one just for uppy?